packages/client/src/components/Foretold__Components__Measurements__Table.re

```diff
@@ -91,6 +97,17 @@ module Helpers = {
     | _ => None
     };
 
+  let getValueText = (measurement: measurement) =>
+    switch (measurement.value) {
+    | Belt.Result.Ok(`FloatCdf(_r)) =>
```

Comment:
wh do you need to add "Belt.Result"; rather than just saying "Ok(..."?

Reply:
Thanks, fixed.